### PR TITLE
12 get /api/articles topic query

### PR DESCRIPTION
**Pull request by joedixon21**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -221,6 +221,32 @@ describe("/api/articles", () => {
                 expect(body.msg).toBe("Not a valid query");
             });
     });
+    test("GET: 200 - takes a topic query and responds with the articles by the topic value specified", () => {
+        return request(app)
+            .get("/api/articles?topic=cats")
+            .expect(200)
+            .then(({ body }) => {
```
Comment (nc-mark-t) [inline __tests__/app.test.js:228]: You will need a length check here

File: __tests__/app.test.js [hunk 2]
```diff
@@ -221,6 +221,32 @@ describe("/api/articles", () => {
...
+            });
+    });
+    test("GET: 404 - responds with 'Not Found' when a topic that doesn't exist is requested", () => {
+        return request(app)
+            .get("/api/articles?topic=pizza")
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Not Found");
+            });
+    });
```
Comment (nc-mark-t) [inline __tests__/app.test.js:249]: What about a topic that has no articles but is valid `ie. paper`

**Review by nc-mark-t — COMMENTED** [review]
Good work, there is one edge case to consider.
